tailwind.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
       colors: {
         primary: {
           400: '#ca322f'
-        }
+        },
+        lightRed: 'rgba(255,210,210,.46)'
```

Comment:
this is fine for now.
But, I will fix this myself and update you about the changes

Reply:
ok